pkg/costmodel/router.go

```diff
@@ -1129,6 +1136,10 @@ func Initialize(additionalConfigWatchers ...ConfigWatchers) {
 		Name: "kubecost_network_internet_egress_cost",
 		Help: "kubecost_network_internet_egress_cost Total cost per GB of internet egress.",
 	})
+	ClusterManagementCostRecorder := prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "kubecost_cluster_management_cost",
```

Comment:
Might be nice to call this `kubecost_cluster_management_hourly_cost` to be more explicit, and to match the `node_cpu_hourly_cost`, etc.